terra-cluster/k8s-pools.tf

```diff
@@ -183,3 +183,31 @@ module "k8s-node-pool-mongodb-v1" {
 
   enable_secure_boot = true
 }
+
+# Elasticsearch Node Pool
+module "k8s-node-pool-elasticsearch" {
```

Comment:
nit: Add version suffix to the module declaration, so it matches the node pool name and variables? Just thinking it will make things cleaner if/when there's ever a v2 node pool 🤷